packages/storybook-test/stories/link/link.stories.tsx

```diff
@@ -146,3 +230,129 @@ export const ExtremelyLongLink: Story = {
     status: { type: [] },
   },
 };
+
+export const Hover: Story = {
+  name: 'Link:hover',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--hover',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :hover state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const Active: Story = {
+  name: 'Link:active',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--active',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :active state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const FocusVisible: Story = {
+  name: 'Link:focus-visible',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--focus-visible',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :focus-visible state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const Current: Story = {
+  name: 'Link (current page)',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--current',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met --current modifier.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const InlineBoxContent: Story = {
+  name: 'Link (rondom image)',
+  args: {
+    children: <ExampleImage />,
+    className: 'nl-link--inline-box-content',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met een image erin en --inline-box-content modifier.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const InlineBoxContentHover: Story = {
+  name: 'Link (rondom image):hover',
+  args: {
+    children: <ExampleImage />,
+    className: 'nl-link--inline-box-content nl-link--hover',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met een image erin, --inline-box-content modifier en :hover state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const InlineBoxContentFocusVisible: Story = {
+  name: 'Link (rondom image):focus-visible',
+  args: {
+    children: <ExampleImage />,
+    className: 'nl-link--inline-box-content nl-link--focus-visible',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met een image erin, --inline-box-content modifier en :focus-visible state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
```

Comment:
Is er geen Story voor Placeholder? Die moet ook met React API, want die vereist nogal bijzondere HTML.

Ik stel me ook voor dat er nog geen unit tests zijn voor al deze dingen.